### ISSUE TITLE
UV Viewer: iframes must have an accessible name

### ISSUE THREAD

**Issue by erinchambers** [CDRH/sparc#394]
When an iframe doesn't have either a `title` attribute or an `aria-label` attribute, screen readers will read the URL instead.

I'd recommend a `title` attribute with something simple and readily understandable, like "Document viewer."

Here's a link to a page with UV Viewer, for convenience: https://salmonpueblo.org/documents/unit/002W#?c=0&m=0&s=0&cv=0&xywh=-20139%2C0%2C43673%2C4392

**Comment by wkdewey** [comment]
Looks like iframes are handled within the UV Viewer javascript rather than in any Rails views

**Comment by wkdewey** [comment]
Could I add `title` to the enclosing div, instead?

**Comment by erinchambers** [comment]
Tested it adding a `title` to the div and it didn't won't work, unfortunately. Keyboard navigation sends focus to the `iframe` element so that's what gets read aloud

I can take a look at Javascript here
